packages/flutter_tools/lib/src/linux/build_linux.dart

```diff
@@ -48,18 +48,18 @@ export PROJECT_DIR=${linuxProject.project.directory.path}
   // Invoke make.
   final String buildFlag = getNameForBuildMode(buildInfo.mode ?? BuildMode.release);
   final Stopwatch sw = Stopwatch()..start();
-  final Process process = await processManager.start(<String>[
-    'make',
-    '-C',
-    linuxProject.makeFile.parent.path,
-    'BUILD=$buildFlag',
-  ]);
   final Status status = logger.startProgress(
     'Building Linux application...',
     timeout: null,
   );
   int result;
   try {
+    final Process process = await processManager.start(<String>[
```

Comment:
This is basically the same as:

```dart
result = await processUtils.stream(<String>[
  'make',
  '-C',
  linuxProject.makeFile.parent.path,
  'BUILD=$buildFlag',
]);
```

Reply:
I think we have a couple of places in the tool where we basically do the same thing. We should clean these up, but I'd like to separate that from the actual bug fix.

I can file a bug to do a cleanup?

Reply:
sgtm

Reply:
Filled https://github.com/flutter/flutter/issues/42261